components/brave_shields/resources/panel/components/tree-list/tree-node.tsx

```diff
@@ -138,23 +92,28 @@ function TreeNode (props: TreeNodeProps) {
         ) : (
           <svg width="12" height="12" fill="currentColor" xmlns="http://www.w3.org/2000/svg"><path fillRule="evenodd" clipRule="evenodd" d="M6 11.25a.75.75 0 0 0 .75-.75V6.75h3.75a.75.75 0 0 0 0-1.5H6.75V1.5a.75.75 0 0 0-1.5 0v3.75H1.5a.75.75 0 0 0 0 1.5h3.75v3.75c0 .414.336.75.75.75Z"/></svg>
         )}
-      </ExpandToggleButton>
+      </S.ExpandToggleButton>
     )
-  }
 
-  const renderBullet = () => {
-    return (
-      <i>
-        <svg width="5" height="5" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
-          <circle cx="1.5" cy="1.5" r="1.5"/>
-        </svg>
-      </i>
-    )
-  }
+    if (isExpanded) {
+      resourcesListElement = (
+        <div ref={treeChildrenBoxRef}>
+          {
+            props.resourceList.map((resource: string, idx) => {
+              return (<ResourceElement
```

Comment:
Prop ordering:

1. key
2. isHost
3. name
4. onTextExpand
5. onPermissionButtonClick
6. permissionButtonTitle

Reply:
There is no key, others fixed